sdk/textanalytics/azure-ai-textanalytics/pom.xml

```diff
@@ -114,7 +114,7 @@
             <configuration>
               <argLine>
                 --add-opens com.azure.ai.textanalytics/com.azure.ai.textanalytics=ALL-UNNAMED
-                --add-opens com.azure.ai.textanalytics/com.azure.ai.textanalytics.implementation.models=com.azure.core
+                --add-exports com.azure.core/com.azure.core.implementation.http=ALL-UNNAMED
```

Comment:
Curious why this was needed?

Reply:
This is a fix for the maven-surefire-plugin. Java repo had merged a PR to having this but cause this error: 
```
java.lang.IllegalAccessError: class com.azure.core.test.TestBase (in unnamed module @0x1e482c2) cannot access class com.azure.core.implementation.http.HttpClientProviders (in module com.azure.core) because module com.azure.core does not export com.azure.core.implementation.http to unnamed module @0x1e482c2
```
This could fix the error and make the test to be able to run again. Otherwise, I won't be able to test.